src/control/client/config_types.go

```diff
@@ -0,0 +1,95 @@
+// (C) Copyright 2018-2019 Intel Corporation.
```

Comment:
should be 2019 as its new file, also want a blank line above to be consistent

Reply:
I've removed the file and consolidated it with config.go.